pkg/platform/local/platform.go

```diff
@@ -714,13 +648,26 @@ func (p *Platform) GetContainerNameByCreateFunctionOptions(createFunctionOptions
 }
 
 func (p *Platform) getContainerHTTPTriggerPort(container *dockerclient.Container) int {
-	ports := container.HostConfig.PortBindings["8080/tcp"]
-	if len(ports) == 0 {
+	functionHostPort := dockerclient.Port(fmt.Sprintf("%d/tcp", abstract.FunctionContainerHTTPPort))
+
+	portBindings := container.HostConfig.PortBindings[functionHostPort]
+	ports := container.NetworkSettings.Ports[functionHostPort]
+	if len(portBindings) == 0 && len(ports) == 0 {
 		return 0
 	}
 
-	httpPort, _ := strconv.Atoi(ports[0].HostPort)
+	port := ""
+	if portBindings[0].HostPort != "" {
```

Comment:
It's possible `len(portBindings) == 0` here

Reply:
you mean `!=` but yeah. as a self convention, I usually compare strings with strings to ensure the reader understand, (as opposed to `len() != 0`, in which he can think it is a list and not a string), but that is just me 🤷

Reply:
No, ==

Reply:
The convention is good imo, and not just you, but you accessed a potentially empty list that's all